tools/serverpod_cli/bin/util/command_line_tools.dart

```diff
@@ -44,6 +44,12 @@ class CommandLineTools {
   }
 
   static Future<void> createTables(Directory dir, String name) async {
+    var _isNetstatAvailable = await existsCommand('netstat');
```

Comment:
Local variables shouldn't have the initial underscore. In this case, it may be cleaner to just put the existsCommand in the if statement. Also, this check should probably be at first when you are running the command, and be overridden by the --force flag. It's also quite simple to manually create the tables, as in the windows version, so instructions should be presented for that rather than the error message.

Reply:
I'm not sure about creating tables.